obsidian_support/conversion/util.py

```diff
@@ -4,6 +4,7 @@
 
 CODE_BLOCK_PATTERN = re.compile(r'(?P<code_block>^[`~]{3,})(?P<language>[a-zA-Z\-]*)?', re.MULTILINE)
 BACKQUOTES_CODE_PATTERN = re.compile(r"`[^\n`]+`")
+HTML_TAG_PATTERN = re.compile(r'<(?P<tag>\S*?)[\s\S]*?>[\s\S]*?</\1>')
```

Comment:
Simple regex like this cannot handle nested html.

e.g.
Given 
`<div> aaa <div> bbb </div> </div>`

AR 
`<div> aaa <div> bbb </div>`

ER 
`<div> aaa <div> bbb </div> </div>`

But, This should cover most cases of HTML that might include conversion exceptions in Markdown. I also want to avoid including HTML parsing libraries like BeautifulSoup. Therefore, I implemented it with simple regex.